Source/Core/DolphinQt/Settings/GameCubePane.cpp

```diff
@@ -105,7 +105,7 @@ void GameCubePane::CreateWidgets()
   for (const auto& entry :
        {std::make_pair(tr("<Nothing>"), ExpansionInterface::EXIDEVICE_NONE),
         std::make_pair(tr("Dummy"), ExpansionInterface::EXIDEVICE_DUMMY),
-        std::make_pair(tr("Broadband Adapter (TAP)"), ExpansionInterface::EXIDEVICE_ETH_TAP)})
+        std::make_pair(tr("Broadband Adapter (TAP)", "virtual network device"), ExpansionInterface::EXIDEVICE_ETH_TAP)})
```

Comment:
I wasn't aware that `tr` had a second parameter. I meant a comment like this: https://github.com/dolphin-emu/dolphin/blob/131f4931fb9fa26a51525a55e7c57265a9812135/Source/Core/DolphinQt/Config/Mapping/MappingWindow.cpp#L309